include/bugle/Translator/TranslateFunction.h

```diff
@@ -132,6 +132,8 @@ class TranslateFunction {
   static bool isPreOrPostCondition(llvm::StringRef fnName);
   static bool isBarrierFunction(TranslateModule::SourceLanguage SL,
                                 llvm::StringRef fnName);
+  static bool isGridBarrierFunction(TranslateModule::SourceLanguage SL,
+                                llvm::StringRef fnName);
```

Comment:
Nitpick:
```suggestion
  static bool isGridBarrierFunction(TranslateModule::SourceLanguage SL,
                                    llvm::StringRef fnName);
```

Reply:
Not sure if this was totally clear, but `llvm::StringRef` should line up with `TranslateModule::SourceLanguage`. Similar for the implementation.